doc/contributing/collaborator-guide.md

```diff
@@ -392,13 +392,21 @@ metadata. Raise a pull request like any other change.
 
 Treat commits that introduce new core modules with extra care.
 
-Check if the module's name conflicts with an existing ecosystem module. If it
-does, choose a different name unless the module owner has agreed in writing to
-transfer it.
+New modules must be added with the `node:` prefix.
```

Comment:
This line is a little ambiguous. Are you saying based on the TSC vote that new modules should be available **only** with the `node:` prefix? If so, I think we should clarify that point (since core modules are already available with the `node:` prefix by default).

Reply:
@cjihrig thanks, incorporated this and your other comments.